src/components/Tool/FixSendNewsletter/GetInvalidNewsletter.graphql

```diff
@@ -20,6 +20,9 @@ query GetInvalidNewsletter($accountListId: ID!) {
       }
     }
   }
+  constant {
+    ...Status
+  }
```

Comment:
I thought about doing that. The reason I didn't was performance. If I understand GraphQL correctly, this should grab only the statuses from the API (which probably takes it from the database) while not requesting the rest of the constants, thus saving 12 more database calls on the back-end. Perhaps it is not worth saving that processing?

Reply:
That is correct. It would be faster to just grab just the statuses, unless UseApiConstants has been called before causing it to be cached. Otherwise, you're correct.

I'm good with you leaving it the way it is if you want.